lxd/network/network_utils.go

```diff
@@ -1032,14 +1032,20 @@ func VLANInterfaceCreate(parent string, vlanDevice string, vlanID string) (bool,
 		return false, nil
 	}
 
+	// set optional gvrp flag
+	gvrpOptionString := ""
```

Comment:
@TimRots can you make gvrp a bool, as mentioned above.

Also I don't think (although admittedly haven't tested it) that this approach will work as you're passing "gvrp on" as a single argument to the `ip` command, but it needs to be two separate arguments.

So instead I suggest making an `args []string` variable and then initialising it to the existing `ip` command args, and then after append the `"gvrp", "on"` to the args slice, and finally then pass the args to the `RunCommand` function as so:

```
	_, err = shared.RunCommand("ip",args...)
```

Reply:
This makes much more sense indeed. Applied your suggestions as requested.